src/interface/tabs.js

```diff
@@ -46,11 +46,21 @@ Tabs.prototype._newTab = function (filepath) {
   tab.on('close', function () {
     self.el.removeChild(tab.el)
 
-    tabs.splice(tabs.indexOf(tab), 1)
+    var activePath = ''
+    var index = tabs.indexOf(tab);
+    tabs.splice(index, 1)
     if (tabs.length === 0) {
       workspace.className = workspace.className + ' tabs-hidden'
+    } else {
+      var nextTab = tabs[index] ? tabs[index] : tabs[index - 1];
```

Comment:
Better written as `tabs[index] || tabs[index - 1]`

Reply:
Good call! I'll make that change!